openpype/hosts/fusion/plugins/create/create_saver.py

```diff
@@ -147,9 +153,10 @@ def _collect_unmanaged_saver(self, tool):
         comp = tool.Comp()
 
         # Allow regular non-managed savers to also be picked up
-        project = legacy_io.Session["AVALON_PROJECT"]
-        asset = legacy_io.Session["AVALON_ASSET"]
-        task = legacy_io.Session["AVALON_TASK"]
+        context = get_current_context()
```

Comment:
Similar issue as here https://github.com/ynput/OpenPype/pull/4932/files#r1192409673 . Asset and task name should be probably taken from the tool attributes instead of from current context?